src/inference/include/openvino/runtime/intel_cpu/properties.hpp

```diff
@@ -63,5 +63,17 @@ static constexpr Property<bool> denormals_optimization{"CPU_DENORMALS_OPTIMIZATI
  */
 static constexpr Property<float> sparse_weights_decompression_rate{"CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE"};
 
+/**
+ * @brief This property used to test accurcay of setting model_distribution_policy to TENSOR_PARALLEL in functional
+ * tests.
+ * @ingroup ov_runtime_cpu_prop_cpp_api
+ *
+ * @code
+ * core.set_property(ov::enable_tensor_parallel(true));
+ * core.set_property(ov::enable_tensor_parallel(false));
+ * @endcode
+ */
+static constexpr Property<bool> enable_tensor_parallel{"ENABLE_TENSOR_PARALLEL"};
```

Comment:
Should be added to python binding, also?

Reply:
In fact, this parameter is for testing only. If it was possible, it would be better to even make it hidden from the user. Until there are no  python tests using this parameter, I don't think it makes sense to promote it to the python API.

Reply:
Move it to internal_properties.hpp then it will be not part of public API.

Reply:
@sunxiaoxia2022 , could you please address the comment above moving the property to `openvino/src/plugins/intel_cpu/src/internal_properties.hpp`?